bin/deployment_archive.js

```diff
@@ -24,14 +24,15 @@ const {join} = require('path');
 const JSZip = require('jszip');
 const zip = new JSZip();
 
-const srcFolder = 'server/dist';
+const destFolder = 'server/dist';
 const distZipFile = process.argv[2] || 'wire-account.zip';
 const ignoreList = ['.DS_Store'];
 
-process.chdir(srcFolder);
+process.chdir(destFolder);
 
 fs.copySync('../package.json', './package.json');
 fs.copySync('../.env.defaults', './.env.defaults');
+fs.copySync('../Procfile', './Procfile');
```

Comment:
i feel like your Procfile is at the wrong level for this command to work

Reply:
It's actually where it should be:
- we first move to `server/dist`
- then go `..` (=> we endup in `server`)
- Procfile is at the root of `server`